pkgs/games/cataclysm-dda/common.nix

```diff
@@ -41,11 +41,6 @@ stdenv.mkDerivation {
 
   postPatch = ''
     patchShebangs .
```

Comment:
Can we update to be specific?

Reply:
I think we can get away with just patching the build script for translation files, since the rest are not invoked during build